config.py

```diff
@@ -109,6 +110,8 @@ class DevelopmentConfig(Config):
     PARTY_USERNAME = os.getenv('PARTY_USERNAME', 'admin')
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD', 'secret')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
+    PARTY_FAKE_DATA = bool(os.getenv('FAKE_PARTY_DATA', False))
+    PARTY_RESPONDENTS_PER_PAGE = os.getenv('PARTY_RESPONDENTS_PER_PAGE', 25)
```

Comment:
This looks like a dupe to me. See line 56

Reply:
It's a dupe to make future changes easier.  Most of the `DevelopmentConfig` members are dupes of `TestingConfig` that just make differences easier to note and make.